test/doc/doxygen/meson.build

```
@@ -0,0 +1,14 @@
+
+
+
+##################################################################################
+# Test the doxy_filter to verify it is flagging comments correctly.
+##################################################################################
+# A standard env for running test scripts under meson.
+doxygen_test_env = {'MESON_SOURCE_DIR':meson.current_source_dir(), 'MESON_BUILD_DIR':meson.current_build_dir()}
+
+# Test the doxy_filter executable create earlier.
+test('doxy_filter',
+    files('filter_test.sh'), args:[doxy_filter.full_path(), 'doxy_filter'],
+    depends: doxy_filter,
```

```suggestion
    files('filter_test.sh'), args:[doxy_filter, 'doxy_filter'],
```

From https://mesonbuild.com/Reference-manual_returned_exe.html#build_tgtfull_path

> NOTE: In most cases using the object itself will do the same job as this and will also allow Meson to setup inter-target dependencies correctly. Please file a bug if that doesn't work for you.

This should work since https://github.com/mesonbuild/meson/commit/11f9425a5e123e7e4bb6296f4453a8e072eb95ed (Meson 0.38.0)